client/interface.go

```diff
@@ -39,6 +39,7 @@ type ContainerAPIClient interface {
 	ContainerTop(ctx context.Context, name string, arguments []string) (types.ContainerProcessList, error)
 	ContainerLogs(ctx context.Context, name string, options types.ContainerLogsOptions) (io.ReadCloser, error)
 	ContainerResize(ctx context.Context, name, height, width string) error
+	ContainerWait(ctx context.Context, name string) (int64, error)
```

Comment:
I think we can use `ContainerWaitOKBody` as return value instead of int64, since the `error` is one of kind error, like `404`, `505` and so on. It can make your code clean. Make senses to you?

Reply:
OK, I'll try it.